operator/pom.xml

```diff
@@ -288,6 +254,27 @@
                     </systemPropertyVariables>
                 </configuration>
             </plugin>
+            
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
Fully specified GAVs are always preferred (minimum of `groupId` and `artifactId`). Although I can find lots of other places in the code already, where we're using antrun without a `groupId` set, so I guess it's not breaking anything? You will inherit jboss-parent's specified version of 1.8 by default, so no need to add a `version` tag in this case.

Reply:
> You will inherit jboss-parent's specified version of 1.8 by default, so no need to add a version tag in this case.

Yes that's what I saw in the ide.  Ideally this task will go away shortly via #20935